### ISSUE TITLE
"Request failed: Cannot find module 'xmldom'"-  when loading data

### ISSUE THREAD

**Issue by retog** [linkeddata/rdflib.js#198]
I included `https://solid.github.io/releases/rdflib.js/rdflib-0.16.2.min.js`as only script. When invoking `fetcher.nowOrWhenFetched` the third argument of the callback function is: Object {ok: false, error: "Request failed: Cannot find module 'xmldom'", status: undefined}.

**Comment by retog** [comment]
The following code

```
<html>
    <head>
        <script src="https://solid.github.io/releases/rdflib.js/rdflib-0.16.2.min.js"></script>
        <script>
            var store = $rdf.graph()
var timeout = 5000 // 5000 ms timeout
var fetcher = new $rdf.Fetcher(store, timeout)

fetcher.nowOrWhenFetched("https://www.w3.org/People/Berners-Lee/card.rdf", function(ok, body, xhr) {
    console.log("now or when fetched");
    if (!ok) {
        console.log("Oops, something happened and couldn't fetch data");
    } else {
        console.log("Got data");
        //do something with data
    }
})
        </script>
    </head>
    <body>
        <div>content</div>
    </body>
</html>
```
Will first output "Got Data" but when trying to access the store it will output "Oops, [...]" and fail

**Comment by dmitrizagidulin** [comment]
@retog Thanks for the heads up!

I've committed a fix for this issue in PR #199. If you'd like to help test it out, do a git fetch, `git checkout switch-to-webpack`, and run `npm run all`.  You should be able to import dist/rdflib.min.js in your script and try it out.

Also, instead of `nowOrWhenFetched()`, try:

```js
var store = $rdf.graph()
var fetcher = new $rdf.Fetcher(store)

fetcher.fetch('https://www.w3.org/People/Berners-Lee/card.rdf')
  .then(function (response) {
    console.log(response)
    console.log(store.toString())  // print the parsed quads
  })
```

**Comment by dmitrizagidulin** [comment]
@retog Oh, also do an `npm install` after checking out the branch.

**Comment by dmitrizagidulin** [comment]
Fixed in v0.16.3, closing.
